trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -164,19 +164,30 @@ subroutine SUBMUSKINGCUNGE(    &
         REAL, intent(INOUT)    :: qdc        ! flow downstream current timestep
         REAL, intent(IN)       :: ql         ! lateral inflow through reach (m^3/sec)
         REAL, intent(IN)       :: Bw         ! bottom width (meters)
+        REAL, intent(IN)       :: ChannK     ! Channel conductivity (m/s)
         REAL, intent(IN)       :: Tw         ! top width before bankfull (meters)
         REAL, intent(IN)       :: TwCC       ! top width of Compund (meters)
         REAL, intent(IN)       :: nCC        ! mannings of compund
+        REAL, intent(IN)       :: ChannK     ! Channel conductivity (m/s)
         REAL, intent(IN)       :: Cs         ! Channel side slope slope
         REAL, intent(IN)       :: So         ! Channel bottom slope %
         REAL, intent(IN)       :: dx         ! channel lngth (m)
         REAL, intent(IN)       :: n          ! mannings coefficient
-        REAL, intent(INOUT)    :: vel        ! mannings coefficient
+        REAL, intent(INOUT)    :: vel        ! velocity (m/s)
+        REAL, intent(INOUT)    :: qloss      ! channel infiltration (m^3/sec)
         INTEGER, intent(IN)    :: idx        ! channel id
         REAL, intent(INOUT)    :: depth      ! depth of flow in channel
 
 !--local variables
         REAL    :: C1, C2, C3, C4
+
```

Comment:
These are old variables from NWM v1.2. I commented them out, and I don't think they are needed here either.

Reply:
yes, please remove.